docs/template-windows.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: windows-vm-template
+  annotations:
+    description: "OCP kubevirt windows VM template"
+    tags: "kubevirt,ocp,template,windows"
+    miq.github.io/kubevirt-is-vm-template: true
+objects:
+- apiVersion: kubevirt.io/v1alpha1
+  kind: OfflineVirtualMachine
+  metadata:
+    name: ${NAME}
+  spec:
+    template:
+      spec:
+        domain:
+          features:
+            acpi: {}
+            apic: {}
+            hyperv:
+              relaxed: {}
+              vapic: {}
+              spinlocks:
+                spinlocks: 8191
+          clock:
+            utc: {}
+            timer:
+              hpet:
+                present: false
+              pit:
+                tickPolicy: delay
+              rtc:
+                tickPolicy: catchup
+              hyperv: {}
+          cpu:
+            cores: ${CPU_CORES}
+          resources:
+            requests:
+              memory: ${MEMORY}
+          os:
```

Comment:
According to https://github.com/kubevirt/kubevirt/pull/650, it is `machine.type` and not `os.machine`.

Reply:
Fixed. I'm not sure about the supported values. "q35" surely is, and it is good enough, so I'm using it.